Xamarin.Forms.Platform.Android/AppCompat/NavigationPageRenderer.cs

```diff
@@ -1002,8 +1002,8 @@ void UpdateToolbar()
 				bar.SetTitleTextColor(textColor.ToAndroid().ToArgb());
 
 			Color navIconColor = NavigationPage.GetIconColor(Current);
```

Comment:
Can `Current` be null at this point?

Reply:
I think this one #9172 takes care of that , it returns the Default color